aws-amplify/docs
# replace amplify framework by amplify

_Issue:_ [#4691](https://github.com/aws-amplify/docs/issues/4691)

_Description of changes:_
Replacing Amplify Framework with Amplify 

_How to test:_
- Go through Amplify docs and check that we don't have "Amplify Framework"
- Meta tags were also updated
![Captura de ecrã 2022-10-15, às 11 37 15](https://user-images.githubusercontent.com/41929489/195982096-a61622be-8efd-4653-a86a-3a05bbd7faa8.png)


By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
